modules/ROOT/pages/install-patches.adoc

```diff
@@ -0,0 +1,111 @@
+= Perform Scheduled Maintenance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Use the following procedure when performing scheduled maintenance for Anypoint Runtime Fabric nodes, such as applying OS patches and security updates. 
+
+== Controller Node Maintenance
+
+The controller nodes are the entry points into your Runtime Fabric cluster, but do not run any application workloads.
+
+To update controller nodes:
+
+. On your upstream TCP load balancer, remove the controller nodes from your load balancing pool, one at a time, during patching and rebooting tasks. 
+. When updates are complete on each controller node, add the node back to the cluster before removing the next one.
+
+== Worker Node Maintenance
+
+To update worker nodes: 
+
+. On a controller node, run the `gravity shell` command.
+. Verify that the status for all nodes is `READY`:
++
+```
+$ sudo kubectl get nodes
```

Comment:
I believe we don't need to run these commands as sudo once we enter gravity shell.
```
ip-172-31-0-91:/$ kubectl get nodes
NAME           STATUS   ROLES    AGE    VERSION
172.31.0.124   Ready    node     2d     v1.13.13-gravitational.0
172.31.0.187   Ready    master   2d     v1.13.13-gravitational.0
172.31.0.219   Ready    node     2d     v1.13.13-gravitational.0
172.31.0.91    Ready    master   2d1h   v1.13.13-gravitational.0
172.31.1.236   Ready    node     2d     v1.13.13-gravitational.0
172.31.1.24    Ready    master   2d     v1.13.13-gravitational.0
172.31.2.112   Ready    master   2d     v1.13.13-gravitational.0
172.31.2.117   Ready    node     2d     v1.13.13-gravitational.0
172.31.3.206   Ready    master   2d     v1.13.13-gravitational.0
172.31.3.23    Ready    node     2d     v1.13.13-gravitational.0
ip-172-31-0-91:/$
```

Reply:
Removed `sudo` - thank you!